### PR TITLE
Closing button is properly centered

### PR DESCRIPTION
**Pull request by jocoso**
### What does this PR do?

Centers the closing buttons on individual modal. 

### Any background context you want to provide?

Screen Shot 2019-06-26 at 3 13 45 PM

Screen Shot 2019-06-26 at 3 38 46 PM

### What are the relevant tickets/cards?

Refs [Pivotal ID #166849627]() 

### Checklist

* [ ] Documentation added for new features/changed endpoints.
* [ ] Added appropriate feature/unit tests.
* [x] Added screenshot to PR description of related front-end updates on **small** screens.
* [ ] Added screenshot to PR description of related front-end updates on **medium** screens.
* [ ] Added screenshot to PR description of related front-end updates on **large** screens.

### CODE REVIEW

**Comment by weerd** [comment]
@jocoso can you make sure that your JSPrettier extension is working and running for your code editor. It looks like something is changing a bunch of files unnecessarily that don't relate to the PR.

File: package.json [hunk 1]
```diff
@@ -64,6 +64,8 @@
     "@dosomething/forge": "^6.9.1",
     "@dosomething/gateway": "^2.0.1",
     "@dosomething/puck-client": "^1.6.0",
+    "acorn-dynamic-import": "^4.0.0",
```
Comment (weerd) [inline package.json]: @jocoso I think these were not supposed to be added to the package.json as dependencies. Are they supposed to be global dependencies for your setup?

File: package.json [hunk 2]
```diff
@@ -73,6 +75,7 @@
     "apollo-link-persisted-queries": "^0.2.2",
     "babel-jest": "^24.8.0",
     "classnames": "^2.2.6",
+    "contentful": "^7.8.0",
```
Comment (weerd) [inline package.json]: This is meant to be a global dependency and not a dependency for the project 😀

File: package.json (hunk 1 above)
Comment (jocoso) [reply package.json]: @weerd  Yes, I noticed there are some weird issues going on with my environment. I don't know how they happened maybe has to do with my npm install or the setting up? Also may be some residual issue for trying to pull the request yesterday.

**Comment by mendelB** [comment]
@jocoso Feel free to merge this one in!

**Pull request merged**